homeassistant/components/cover/__init__.py

```diff
@@ -224,6 +235,10 @@ def state_attributes(self):
         if current_tilt is not None:
             data[ATTR_CURRENT_TILT_POSITION] = self.current_cover_tilt_position
 
+        current_tilt_state = self.tilt_state
+        if current_tilt_state is not None:
+            data[ATTR_CURRENT_TILT_STATE] = self.tilt_state
```

Comment:
Why add the intermediate local variable if you are not going to use it consistently? I don't think you need the intermediate variable here.